commands/output/output.go

```diff
@@ -170,18 +173,30 @@ func printAnalyses(r *review.Review) {
 	fmt.Println("  analyses: ", r.GetAnalysesMessage())
 }
 
-// printComments prints all of the comments for the review, with snippets of the preceding source code.
-func printComments(r *review.Review) error {
-	fmt.Printf(commentSummaryTemplate, len(r.Comments))
-	for _, thread := range r.Comments {
-		err := showThread(r, thread)
+// printCommentsWithIndent prints all of the comment threads with the given indent before each line.
+func printCommentsWithIndent(repo repository.Repo, c []review.CommentThread, indent string) error {
+	for _, thread := range c {
+		err := showThread(repo, thread, indent)
 		if err != nil {
 			return err
 		}
 	}
 	return nil
 }
 
+// PrintComments prints all of the given comment threads.
+func PrintComments(repo repository.Repo, c []review.CommentThread) error {
+	fmt.Printf(detachedCommentSummaryTemplate, len(c))
+	return printCommentsWithIndent(repo, c, "")
```

Comment:
Why no indent here? Seems like a strange choice.

Will that make the output something like this? If so seems strange.
```interactive
Comments: (1 threads):
comment: xxxxxxx
author: xxxxxxx
time:   xxxxxx
status: xxxxxxx
```

Reply:
Good point; there should be an indent of at least a couple spaces.

Done